connectors/sources/sharepoint_online.py

```diff
@@ -1148,9 +1254,10 @@ async def get_docs(self, filtering=None):
                 site_collection["siteCollection"]["hostname"],
                 self.configuration["site_collections"],
             ):
-                yield site, None
+                access_control = await self._site_access_control(site)
+                yield self._decorate_with_access_control(site, access_control), None
```

Comment:
So in essence every document within site is gonna have same permissions?

Out of curiosity, do we know what's the storage impact of this?

Reply:
Yes, it's reduced now as we're only adding groups and not individual emails/usernames (only if a user is a person, lol SPO?), if the same information can be derived from an identity being part of a group.